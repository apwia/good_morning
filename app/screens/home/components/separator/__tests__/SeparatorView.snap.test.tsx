import 'react-native';
import React from 'react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
import SeparatorView from '../SeparatorView';

it('renders correctly', () => {
 const {toJSON} = renderer.create(<SeparatorView />);
 expect(toJSON()).toMatchSnapshot();
});