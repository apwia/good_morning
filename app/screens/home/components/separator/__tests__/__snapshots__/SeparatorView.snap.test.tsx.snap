// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    {
      "borderBottomColor": "#737373",
      "borderBottomWidth": 0.5,
      "marginVertical": 8,
    }
  }
/>
`;
